pandas/tests/io/test_sql.py

```diff
@@ -855,11 +848,13 @@ class _TestSQLApi(PandasSQLTest):
     flavor = "sqlite"
     mode: str
 
-    def setup_connect(self):
-        self.conn = self.connect()
-
     @pytest.fixture(autouse=True)
```

Comment:
I _think_ this is redundant with `setup_method`

Reply:
If I remove the line, `@pytest.fixture(autouse=True)`, I get the following error:
```
TypeError: _TestSQLApi.setup_method() missing 1 required positional argument: 'types_data'
```